src/ui/map.js

```diff
@@ -2847,7 +2847,12 @@ class Map extends Camera {
             const elevationChange = Math.abs(currentElevation - newElevation);
 
             if (elevationChange > AVERAGE_ELEVATION_EASE_THRESHOLD) {
-                this._averageElevation.easeTo(newElevation, timeStamp, AVERAGE_ELEVATION_EASE_TIME);
+                if (this._isInitialLoad) {
+                    this._averageElevation.jumpTo(newElevation);
+                    return applyUpdate(newElevation);
```

Comment:
There's a small amount of code duplication with this and the block that follows, but after some different permutations of this, DRYing it up seemed counterproductive to maintainability and clarity.